api/libproject/devstart.rb

```diff
@@ -1557,6 +1602,11 @@ def migrate_workbench_data()
   end
 end
 
+def get_fc_config(project)
```

Comment:
We can now rewrite the next 2 functions in terms of this one, to avoid code duplication.

Reply:
Good catch - done.